addons/mail/static/src/core/web/chat_window_service.js

```diff
@@ -0,0 +1,21 @@
+/* @odoo-module */
+
+import { ChatWindowService } from "@mail/core/common/chat_window_service";
+
+import { patch } from "@web/core/utils/patch";
+
+patch(ChatWindowService.prototype, "mail/core/web", {
+    async close() {
+        const _super = this._super.bind(this); // limitation of class.js
```

Comment:
not class.js but patch?

Reply:
there is an await later

Reply:
class.js is the legacy odoo class system, we don't use that anymore

Reply:
removed this misleading comment, and also took opportunity to add suffix `_patch` in new file